JavaScript1/Week2/homework.md

```diff
@@ -6,200 +6,157 @@
 - Go through the review of [last week](../Week1/review.md) (Work in progress, update this week :wrench:)
 - Go through the review of [this week](../Week2/review.md) (work in progress, update this week :nut_and_bolt:)
 
-## Step 2: JavaScript
-> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference
 
-1. Create a function that takes 3 arguments and returns the sum of the three arguments.
+## Step 2: Javascript warmup part one
+Just like last time, lets warmup our brain!
+Go back to FreeCodeCamp, start where you left of and finish the rest of the Basic JavaScript challenges. We know this seems like a lot, but the tasks are not so big, so hang in there! 
```

Comment:
Not sure if typos are even worth mentioning? Should say "..where you left off.."

Reply:
Yes, we should do typos aswell.